R/pipgd_lorenz.R

```diff
@@ -374,6 +400,22 @@ pipgd_lorenz_curve <- function(
 }
 
 
+# # Example 1:
```

Comment:
I Think it is not working because you need the option `complte = TRUE`. @zander-prinsloo, please confirm.